readme.md

```diff
@@ -22,7 +22,9 @@ console.log(await npmUser('sindresorhus'));
 {
 	name: 'Sindre Sorhus',
 	avatar: 'https://gravatar.com/avatar/d36a92237c75c5337c17b60d90686bf9?size=496',
```

Comment:
This now returns `/npm-avatar/...`, which is only valid when prefixed with `https://www.npmjs.com`. Should that added to the result?

Reply:
It should be an absolute URL, yes.